supporting_content/columns/skufamily.md

```diff
@@ -0,0 +1,29 @@
+# Column: SKU Family
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                   | Column |
+| -------- | -------------------------- | ------ |
+| AWS      | CUR                        |        |
```

Comment:
For [AWS](https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html#product-details-I-instancetypefamily), `product/instanceTypeFamily` is close if not identical, and I think a good example to add in the supporting content.